codefest/settings.py

```diff
@@ -220,4 +222,16 @@
 CELERY_TASK_SERIALIZER = 'json'
 CELERY_BROKER_URL = 'amqp://localhost'
 if not DEBUG:
-    CELERY_BROKER_URL=config('CLOUDAMQP_URL')
\ No newline at end of file
+    CELERY_BROKER_URL=config('CLOUDAMQP_URL')
+
+
+ASGI_APPLICATION = 'codefest.asgi.application'
+CHANNEL_LAYERS = {
+    'default': {
+        # 'BACKEND': 'channels_redis.core.RedisChannelLayer',
+        # 'CONFIG': {
+        #     "hosts": [('127.0.0.1', 6379)],
+        # },
+        'BACKEND': 'channels.layers.InMemoryChannelLayer',
```

Comment:
Will this work when we deploy on heroku dynos @krashish8?

Reply:
Not sure about it, but it seems that this might require some additional configurations, maybe in the Procfile?

Reply:
I mean the dyno restarts every 5 mins. So, the RAM is cleared too. I think we can use redis (Heroku has a add-on for that).

Reply:
Yes, I agree.

I think the dyno restarts every 30 mins, but we have Uptime Robot to counter that (just a workaround). Well, the Uptime Robot is still active, and Amri bhaiya's dyno hours are consumed every month 🤭

Reply:
I guess this part is only for development purposes. :thinking: